### PR TITLE
Update section on how to add pull request to project board

### LINKED ISSUE #2670: Update the section on how to add a pull request to the project board in CONTRIBUTING.md
### Dependencies
- [x] https://github.com/hackforla/website/issues/2641
- [x] https://github.com/hackforla/website/issues/2395 

### Overview
As developers, we need to make sure our guide on how to contribute is up to date. In this issue, we will update the section on how to add a new pull request to the Project Board.

### Action Items
- [x] In the section on "How to add your pull request to the project board" in CONTRIBUTING.md, please update the following sentence in the instructions:
```
We are in the process of automating newly created pull requests being added to the Project Board.
```
to 
```
There is an automation for adding newly created pull requests to the Project Board.
```

- [x] Commit and push changes to your branch on your forked repository
- [x] In CONTRIBUTING.md on your remote branch, check that your changes have been made

### Resources/Instructions
- [CONTRIBUTING.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [How to add a pull request to the project board](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#v-complete-pull-request-5-how-to-add-a-pull-request-to-the-project-board)

### PR DESCRIPTION
**Pull request by thedvo**
Fixes #2670 

### What changes did you make and why did you make them ?

  - Update "How to add your pull request to the project board" section in CONTRIBUTING.md
  - Changed the sentence noted in the instructions
  

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

Updating markdown file in website repository. No visual changes to the website.

### CODE REVIEW

**Comment by arcan9** [comment]
Review ETA: EOD 04/05
Availability: after 6pm PST, M-F

**Comment by kwangric** [comment]
Review ETA: 4/5 7pm PDT
Availability: after 5pm PDT

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ The following is a set of guidelines for contributing to the website repository,
       - [**2.3.d Issues for Hacktoberfest contributors (Front-End/Back-End)**](#23d-issues-for-hacktoberfest-contributors-front-endback-end)
       - [**2.3.e What if you see bugs/errors that are not connected to an issue?**](#23e-what-if-you-see-bugserrors-that-are-not-connected-to-an-issue)
     - [**2.4 Claiming an Issue**](#24-claiming-an-issue)
-      - [**2.4.a Assign & Unassign yourself to this issue**](#24a-assign--unassign-yourself-to-this-issue)
+      - [**2.4.a Assign \& Unassign yourself to this issue**](#24a-assign--unassign-yourself-to-this-issue)
```
Comment (arcan9) [inline CONTRIBUTING.md]: Just wondering why you added a `\` (backslash) in front of the `&` (ampersand) on this line? Same thing on line 45.

**Review by arcan9 — DISMISSED** [review]
Nice work @thedvo! The PR is created correctly (branch, linked issue, documentation). You made the correct changes. I just have an additional comment below.

File: CONTRIBUTING.md (hunk 1 above)
Comment (thedvo) [reply CONTRIBUTING.md]: Thanks for reviewing @arcan9! That's strange I'll check into that and remove those.

Comment (thedvo) [reply CONTRIBUTING.md]: @arcan9 I've removed the backslashes and pushed the changes. Let me know if there's any issues.

**Review by kwangric — APPROVED** [review]
The changes look good and covers everything. Issue was linked and branches were set up correctly as well. Great work, Dan!

File: CONTRIBUTING.md (hunk 1 above)
Comment (arcan9) [reply CONTRIBUTING.md]: @thedvo I've checked. Thanks for removing them! The links still work just fine.

**Pull request merged**